NEWS.md

```diff
@@ -1,3 +1,24 @@
+## 19.3.0 2020-07-28
+
+* Add 'Aged to lost' status to allowed item statuses list (MODINVSTOR-503)
+* Make migratePrecedingSucceedingTitles.sql idempotent (MODINVSTOR-541)
+* Separate filtering from response generation in oai-pmh view (MODINVSTOR-536)
+* Item record. Effective call number (item), eye readable. Search on exact data in Call number data element (MODINVSTOR-481)
+* Holdings record. Call number, eye readable. Search on exact data in Call number data element (MODINVSTOR-480)
+* Search queries without database index (MODINVSTOR-472)
+* Upgrade to raml-module-builder (RMB) 30.2.4 (MODINVSTOR-532)
+* Expand oai-pmh view with additional fields (MODINVSTOR-498)
+* OAI-PMH view doesn't respond in Bug Fest (MODINVSTOR-527)
+* Change permissions location for oai-pmh view (MODINVSTOR-524)
+* Add Index for Instances Full-text Subjects Search (MODINVSTOR-499)
+* Postgres requires special permissions to disable triggers for migrations (MODINVSTOR-476)
+* Fixing inconsistent hit counts (totalRecords) estimation (MODINVSTOR-519)
+* Bulk download Instance UUIDs OOM (MODINVSTOR-465)
```

Comment:
This list contains 2 issues that already have been released in b19.2:
* MODINVSTOR-476 (fixed in mod-inventory-storage 19.2.2)
* MODINVSTOR-465 (fixed in mod-inventory-storage 19.2.1)

Do we need to mention them in 19.3.0 again, or is it sufficient to list them in 19.2.2 or 19.2.1 release notes?

Reply:
Personally, I think we should because this is a feature release. I don't think folks should be expected to read the release notes for patch releases to a previous feature release, it should be sufficient to read feature releases only.

This probably comes from my mental model that patch releases are like separate tracks off to one side of the main sequence of feature releases.

This is purely my personal opinion, I don't believe FOLIO has any opinion.